devops/scripts/register_bundle_with_api.sh

```diff
@@ -159,26 +172,39 @@ else
     usage
   fi
 
-  case "${BUNDLE_TYPE}" in
+  case "${bundle_type}" in
     ("workspace") tre_get_path="api/workspace-templates" ;;
     ("workspace_service") tre_get_path="api/workspace-service-templates" ;;
     ("user_resource") tre_get_path="/api/workspace-service-templates/${workspace_service_name}/user-resource-templates";;
+    ("shared_service") tre_get_path="/api/shared-service-templates";;
   esac
 
   echo -e "Server Response:\n"
   eval "curl -X 'POST' ${tre_url}/${tre_get_path} -H 'accept: application/json' -H 'Content-Type: application/json' -H 'Authorization: Bearer ${access_token}' -d '${payload}' ${options}"
   echo -e "\n"
 
-  if ${verify}
-  then
+  if [[ "${verify}" = "true" ]]; then
```

Comment:
Why do you need "true"? Isn't Boolean somethin that works without the string notation?

Reply:
What do you mean by a boolean? There are no booleans in Bash. You can declare a string variable, or a number, and compare it.
In this case, the "verify" was already declared as a string variable, and all I did was to make the if statements in the script more consistent, and do an explicit comparison (as per recommended, for example, here: https://stackoverflow.com/a/21210966/3488743)